Docs/source/Overview/design-requirements.rst

```diff
@@ -15,6 +15,8 @@ The BPS should have the following components and features:
         * the voltage of a module exceeds the safe range
         * the current through the cells exceeds the safe range
         * the temperature exceeds the maximum specified in the datasheet
+        * an open wire is detected
+        * the BPS RTOS stops functioning properly
```

Comment:
we also trip if there is a hard fault, or if we drop too many spi packets from the voltage and temperature minions in a row